ecoscope/io/__init__.py

```diff
@@ -1,15 +1,29 @@
 from ecoscope.io import earthranger, eetools, raster, utils
 from ecoscope.io.earthranger import EarthRangerIO
-from ecoscope.io.async_earthranger import AsyncEarthRangerIO
 from ecoscope.io.utils import download_file
 
 __all__ = [
     "earthranger",
     "EarthRangerIO",
-    "AsyncEarthRangerIO",
     "download_file",
     "earthranger_utils",
     "eetools",
     "raster",
     "utils",
 ]
+
+try:
+    from ecoscope.io.async_earthranger import AsyncEarthRangerIO
+
+    __all__ = [
+        "earthranger",
+        "EarthRangerIO",
+        "AsyncEarthRangerIO",
+        "download_file",
+        "earthranger_utils",
+        "eetools",
+        "raster",
+        "utils",
+    ]
```

Comment:
```suggestion
    __all__ += ["AsyncEarthRangerIO"]
```
?

Reply:
I had tried `__all__.append()` but ran into issues with the linter - this seems to both work and keep the linter happy. Thanks!